### ISSUE TITLE
Task 3: Evaluate performance of pre-trained network on node-level tasks

### ISSUE THREAD

**Issue by eeishaan** [eeishaan/pretrain-gnns#3]
1. Load the weights of the pre-trained model. Both the stages have been completed for this model.
1. Fine-tune the model in a supervised fashion on a node-level task such as community detection. Doesn't matter which task. Just search and pick one. But, it should be node-level task. Could be link prediction or node-label classification.
1. Report performance on test-set
